### DISCUSSION TITLE
Adding more useful, data attached to markdown while parshing

### DISCUSSION THREAD

**Discussion by q1b** [withastro/roadmap#233] [Proposal]
While making my own website, I find it useful to have some, stuff for a blog which is quite awesome, like **reading time for a post**,

I am interested in implementing it, into astro but experience lacks with codebase. 

this is the utility which we can integrate into our astro core, https://github.com/syntax-tree/hast-util-reading-time to support this feature, So it will be a meta data for a post

**Comment by delucis** [comment]
Noting that the Astro docs do include a recipe for adding reading time: https://docs.astro.build/en/recipes/reading-time/
